src/plugins/gui/providers/ioniaProvider.ts

```diff
@@ -391,12 +398,25 @@ export const makeIoniaProvider: FiatProviderFactory<IoniaMethods> = {
 
           return true
         },
+        hideCard: async cardId => {
+          const set = new Set(hiddenCardIds)
+          set.add(cardId)
+          hiddenCardIds = Array.from(set)
+          await store.setItem(STORE_HIDDEN_CARDS_KEY, wasStoreHiddenCards(hiddenCardIds))
+        },
         async getGiftCards() {
           const giftCardsResponse = await fetchGetGiftCards({
             headers: userAuthenticatedFetchOptions.headers
           })
           const { Data: giftCards } = giftCardsResponse
-          return giftCards
+
+          const out: { cards: GiftCard[]; archivedCards: GiftCard[] } = { cards: [], archivedCards: [] }
```

Comment:
archivedCards are defined but unused

Reply:
Ah, yeah, I was thinking about using them in the UI later. Like we could show text-button "Show Archived Cards" if we see the length > 0 for archivedCards